scripts/data/experiment.json

```diff
@@ -156,4 +156,26 @@
     "endDate": "2016-02-02",
     "lastEdited": "2016-02-04",
     "structure": []
+}, {
+    "id": "test9",
+    "title": "Lookit Test",
+    "description": "It's a test",
+    "state": "Active",
+    "eligibilityCriteria": "age >= 3 years and age < 6 years",
+    "structure": [
+        {
+            "type": "lookit-overview",
```

Comment:
In the new proposed schema, there are three possible kinds of frame names, and all "actual" frames must be prefixed with `exp-*`.

If you want this frame to deviate from that convention... I'm not sold on all frame types needing the special prefix, but what design we choose will affect this frame name.